app/models/world_location.rb

```diff
@@ -53,16 +56,21 @@ def initialize(data)
     @data = data
   end
 
+  def slug
+    self.details["slug"]
+  end
+
   def ==(other)
-    other.is_a?(self.class) && other.slug == self.slug
+    other.is_a?(self.class) && other.slug == slug
   end
 
   def_delegators :@data, :title, :details
-  def_delegators :details, :slug
+  # def_delegators :details, :slug
```

Comment:
This has been removed,.. see https://github.com/alphagov/smart-answers/blob/3c8ff7a363617d959145b70ac58a7d934757b1ec/app/models/world_location.rb#L61